src/main/scala/no/ndla/articleimport/service/converters/LeafNodeConverter.scala

```diff
@@ -80,11 +80,10 @@ trait LeafNodeConverter {
         .toH5PEmbed(cont.nid) match {
         case Success(html) =>
           el.append(s"<section>$html</section>")
-          Success(
-            (cont.copy(content = jsoupDocumentToString(el), metaDescription = defaultMetaDescription), importStatus))
+          Success((cont.copy(content = jsoupDocumentToString(el)), importStatus))
         case Failure(ex) =>
           if (el.text().length > 0) {
-            Success(cont.copy(content = jsoupDocumentToString(el), metaDescription = defaultMetaDescription),
```

Comment:
ref [denne](https://trello.com/c/KsxjasR4/451-metatekstproblem-relasjoner-video-h5p-noder). Skal ikke feltet heller settes til en tom streng?

Reply:
I følge [denne ](https://trello.com/c/NDUAVGRF/624-artikler-som-genereres-fra-h5p-m%C3%A5-f%C3%A5-med-metabeskrivelse) skal det settes metabeskrivelse fra ingressen til artikkelen. Det er bedre at metabeskrivelsen som faktisk finnes brukes i stedet for å sette Beskrivelse mangler rått, eller tom streng.